library/src/main/java/com/schibsted/spain/barista/assertion/BaristaAssertions.kt

```diff
@@ -23,4 +25,8 @@ object BaristaAssertions {
         }
     }
 
+    @JvmStatic
+    fun assertActivityIsFinishing(activity: Activity){
```

Comment:
Hmm this feels like a new feature different from what the PRs says.
I'm not sure it adds much value, since it's just a 1-line method for something the activity instance already has. I mean, it's not hiding any complex operations with a better name, it would be OK to just write `assertTrue(activity.isFinishing)` in a real test IMHO. Don't you think?

Reply:
Maybe a extension function of barista rule?

`rule.assertActivityIsFinishing()` @Sloy ?

Reply:
I don't think Barista adds any value  there. We provide shortcuts for things with a complex syntax, like most Espresso methods. But since this doesn't use espresso synchronization or anything, can be asserted with any simple assertion library like junit, assertJ, truth...